(DOCSP-44489) stakeholder feedback (#106)
mongodb/docs-atlas-architecture: source/hierarchy.txt

   * - (Optional) Paying Organization
     - One organization can be a paying organization for other
       organization(s). A paying organization lets you set up 
       :atlas:`cross-organization billing <billing/#std-label-cross-org-billing>` 
       to share a billing subscription across multiple organizations. 
       To learn more about setting up your paying organization when 
       you establish the |service| subscription, see :atlas:`Manage Billing <billing>`. 

>        A paying organization lets you set up 
>        :atlas:`cross-organization billing <billing/#std-label-cross-org-billing>` 
>        to share a billing subscription across multiple organizations. 

Reviewer: Duplicate

```suggestion

```

>        In order to enable cross-organization billing, the user performing 
>        the action must have an Org Owner or Billing Admin role of both 
>        organizations that they wish to link.

Reviewer: Should we link to role descriptions?

```suggestion
       To enable cross-organization billing, you must have an Org Owner or Billing Admin role for both 
       organizations that you want to link.
```

       A paying organization is common for large enterprises with many
       {+BU+}\s or departments that operate independently but where the
       contract or bill is owned by a central authority.

   * - Organization
     - An organization can contain many projects under it and provides
       a container to apply shared integration and security settings. If 

>        you are managing multiple {+service+} organizations, 

Reviewer: ```suggestion
       you manage multiple {+service+} organizations, the
```

       :atlas:`Atlas Federation Management Console <atlas-federated-authentication>` 
       allows users with an Org Owner role to manage |idp|s for |sso|, 
       then link them to multiple organizations. 
       An organization often maps to a {+BU+} or department within a
       company. The built-in {+service+} Cost Explorer aggregates cloud

>        spend at the organization level and breaks out line items at the 
>        project and {+cluster+} level below it. You can customize further

Reviewer: ```suggestion
       spend at the organization-level and breaks out line items at the 
       project-level and {+cluster+}-level below it. You can customize further
```

       by leveraging the billing API.

   * - Project
     - Security configuration for the data plane (including database
       clusters, network security, and other data services) occurs at the 

>        project level. A project often maps to an application and environment 

Reviewer: ```suggestion
       project-level. A project often maps to an application and environment 
```

       (for example: Customer Portal app - Production environment). For each 
       project, based on the selected cloud provider, there is one dedicated 

>        VPC/VNet per region in AWS and Azure. 

Reviewer: ```suggestion
       VPC or VNet per region in AWS and Azure. 
```

Generally, we recommend a paying organization that is managed
centrally, and one organization for each {+BU+} or department that is
linked to the paying org. Then, create a project with one {+cluster+}

> each for your lower (dev/test) and upper environments; cluster(s) can 

Reviewer: ```suggestion
each for your lower (dev or test) and upper environments; you can create clusters
```

> be created in said projects. To learn more, see the following information 

Reviewer: ```suggestion
in these projects. To learn more, see the following information 
```

for :ref:`<project-hierarchy-1>`.

If you will easily hit the 250 project limit per organization, we
recommend creating one organization per environment, such as one each
for lower and upper environments, or one each for dev, test, staging,

> and prod. This has an added benefit of additional isolation. You can also 

Reviewer: ```suggestion
and production. This setup has the benefit of additional isolation. You can also 
```

> increase the limits. See :atlas:`<atlas-limits>` for more informaiton.

Reviewer: ```suggestion
increase the limits. To learn more, see :atlas:`<atlas-limits>`.
```

{+Cluster+} Hierarchy
`````````````````````

> To maintain isolation between environments, we recommend deploying 

Reviewer: ```suggestion
To maintain isolation between environments, we recommend that you deploy 
```
